### PR TITLE
REFACTOR: EDB configuration example for DCIR setup

### PR DESCRIPTION
**Pull request by gkorompi**

### CODE REVIEW

**Review by hui-zhou-a — COMMENTED** [review]
Please wait until https://github.com/ansys/pyedb/pull/903 is merged. Then, merge main into this branch.

There are two more thing need  to be done to make this example show up in documentation.

update doc/source/conf.py and exmaples/use_configuration/index.rst

File: examples/use_configuration/dcir.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+from ansys.aedt.core import Hfss3dLayout
+from ansys.aedt.core.downloads import download_file
+
+from pyedb import Edb
+
+AEDT_VERSION = "2024.2"
+NG_MODE = False
+
+# -
```
Comment (SMoraisAnsys) [inline examples/use_configuration/dcir.py:19]:
```suggestion
# +
import json
import os
import tempfile

from ansys.aedt.core import Hfss3dLayout
from ansys.aedt.core.downloads import download_file

from pyedb import Edb

AEDT_VERSION = "2024.2"
NG_MODE = False

# -
```

**Review by SMoraisAnsys — CHANGES_REQUESTED** [review]
As Hui mentionned, you should also update how this file interacts with the configuration by modifying RST files and also `conf.py` if required you want to add a thumbnail manually.

File: examples/use_configuration/dcir.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
```
Comment (SMoraisAnsys) [inline examples/use_configuration/dcir.py:18]:
```suggestion
# -
```

**Review by SMoraisAnsys — APPROVED** [review]
LGTM

**Comment by SMoraisAnsys** [comment]
@hui-zhou-a Can you have a look at this PR now that #903 is merged ?

**Review by hui-zhou-a — APPROVED** [review]
LGTM

**Pull request merged**